tests/test_cases/test_vhdl_zerovector/test_vhdl_zerovector.py

```diff
@@ -0,0 +1,36 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import cocotb
+from cocotb.triggers import Timer
+import pytest
+
+
+@cocotb.test()
+async def test_long_signal(dut):
+    """ Write and read a normal signal (longer than 0)."""
+    dut.data_in <= 0x5
+    await Timer(1, "ns")
+    assert dut.data_out == 0x5, "Failed to readback dut.data_out"
+
+
+@cocotb.test(expect_error=AttributeError if cocotb.SIM_NAME.lower().startswith(("ghdl", "xmsim", "ncsim")) else ())
+async def test_read_zero_signal(dut):
+    """ Read a zero vector. It should always read 0."""
+    assert dut.Cntrl_out == 0, "Failed to readback dut.Cntrl_out"
+
+
+@cocotb.test(expect_error=AttributeError if cocotb.SIM_NAME.lower().startswith(("ghdl", "xmsim", "ncsim")) else ())
```

Comment:
To prevent riviera and Active-hdl to fail, I can add them here. What would be their name? 
```suggestion
@cocotb.test(expect_error=AttributeError if cocotb.SIM_NAME.lower().startswith(
    ("ghdl", "xmsim", "ncsim", "riviera", "activehdl")) else ())
```

```suggestion
@cocotb.test(expect_error=AttributeError if cocotb.SIM_NAME.lower().startswith(
    ("ghdl", "xmsim", "ncsim", "riviera", "activehdl")) else ())
```

Reply:
`("riviera", "aldec")`